web/packages/teleport/src/AccessRequests/LockedAccessRequests/LockedAccessRequests.tsx

```diff
@@ -0,0 +1,161 @@
+
+import { CtaEvent } from 'teleport/services/userEvent';
+
+import step1 from './assets/step1.png';
+import step2 from './assets/step2.png';
+import step3 from './assets/step3.png';
+
+export function LockedAccessRequests() {
+  const CTAButton = (
+    <ButtonLockedFeature height="36px" event={CtaEvent.CTA_ACCESS_REQUESTS}>
+      Unlock Access Requests With Teleport Enterprise
+    </ButtonLockedFeature>
+  );
+
+  return (
+    <FeatureBox>
+      <FeatureHeader alignItems="center" justifyContent="space-between">
+        <FeatureHeaderTitle>Just In Time Access Requests</FeatureHeaderTitle>
+        <Box>{CTAButton}</Box>
+      </FeatureHeader>
+      <Card
+        m="auto"
+        p="24px"
+        pb="44px"
+        as={Flex}
+        flex="0 0 auto"
+        flexDirection="column"
+        width="max-content"
+      >
+        <Box width="100%" textAlign="left">
+          <Text typography="h4" bold>
+            Access Requests Flow
+          </Text>
+          <Text typography="subtitle1" mb={5}>
+            To learn more about access requests, take a look at&nbsp;
+            <Link
+              color="text.secondary"
+              href="https://goteleport.com/features/access-requests/"
+              target="_blank"
+            >
+              Teleport Documentation.
+            </Link>
+          </Text>
+        </Box>
+        <Flex gap="24px" flexWrap="wrap" style={{ position: 'relative' }}>
+          <StepCard src={step1} step="1">
+            Bob can select the resources he needs to access or request the role
+            <span style={{ fontWeight: 'bold' }}> dbadmin </span>
+            in the Web UI or CLI
+          </StepCard>
+          <StepCard src={step2} step="2">
+            Chatbot will notify both Alice and Ivan
+          </StepCard>
+          <StepCard src={step3} step="3">
+            Alice and Ivan can review and approve request using Web UI or CLI
+          </StepCard>
+          <Box
+            css={`
+              position: absolute;
+              bottom: 0;
+              left: 50%;
+              min-width: 600px;
+              transform: translate(-50%, 50%);
+            `}
+          >
+            {CTAButton}
+          </Box>
+        </Flex>
+      </Card>
+    </FeatureBox>
+  );
+}
+
+type StepCardProps = {
+  src: string;
+  step: string;
+  children: React.ReactNode;
+};
+
+function StepCard({ src, step, children }: StepCardProps) {
+  return (
+    <StepCardContainer textAlign="left">
+      <ImageContainer>
+        <Image
+          css={`
+            display: block;
+          `}
+          src={src}
+        />
+      </ImageContainer>
+      <Box mt={4} width="100%">
+        <Text fontSize="3" bold>
+          Step {step}
+        </Text>
+      </Box>
+      <Box width="100%">
+        <Text mt="1">{children}</Text>
+      </Box>
+    </StepCardContainer>
+  );
+}
+
+const StepCardContainer = styled(Flex)(
+  ({ theme }) => `
```

Comment:
Nit: I prefer not to return the style template from a function, so the styled components plugin can recognize the template and can provide autocompletion for the property names (for some reason it doesn't work when a template is returned from a function).
Compare 
<img width="419" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/5a3e6af9-f88f-4e85-9d07-cf25597cf3f3">
vs 
<img width="419" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/b143a04a-ba26-4079-8595-5f39a5a29ff3">